package.json

```diff
@@ -49,7 +49,7 @@
     "@types/pretty-ms": "^3.0.0",
     "@types/request": "^2.0.7",
     "@types/sinon": "^4.0.0",
-    "clang-format": "^1.0.33",
+    "clang-format": "^1.0.41",
```

Comment:
Why does this module depend on `clang-format` in the first place?

Reply:
Not sure when dependency was added -- but I was able to remove this without problem.